### PR TITLE
:wrench: Streamline environment variables

### PR DESCRIPTION
**Pull request by stevenbal**
issue: https://github.com/open-zaak/open-zaak/issues/1629

**Changes**
* :fire: Remove unused sniplates template
* :wrench: Add missing DEFAULT_FROM_EMAIL envvar
* :wrench: Add missing CACHE_OIDC envvar
* :wrench: Remove DB_ENGINE envvar
* :wrench: Add missing Django CORS headers settings
* :wrench: Bring logging settings in line with Open Zaak
* :wrench: Bring Elastic APM settings in line with Open Zaak/Open Notificaties
* :wrench: Add USE_X_FORWARDED_HOST setting
* :wrench: Replace os.getenv with config

### CODE REVIEW

File: src/openklant/templates/sniplates/form.html
Comment (annashamray) [inline src/openklant/templates/sniplates/form.html:1]: Could you make an issue to remove it from our default project?

Comment (stevenbal) [reply src/openklant/templates/sniplates/form.html:1]: https://taiga.maykinmedia.nl/project/sergei-default-project/us/44?no-milestone=1

File: src/openklant/conf/base.py [hunk 1]
```diff
@@ -454,6 +490,19 @@
...
+CORS_ALLOW_ALL_ORIGINS = config("CORS_ALLOW_ALL_ORIGINS", default=False)
+CORS_ALLOWED_ORIGINS = config("CORS_ALLOWED_ORIGINS", split=True, default=[])
+CORS_ALLOWED_ORIGIN_REGEXES = config(
+    "CORS_ALLOWED_ORIGIN_REGEXES", split=True, default=[]
+)
+# Authorization is included in default_cors_headers
+CORS_ALLOW_HEADERS = list(default_cors_headers) + config(
+    "CORS_EXTRA_ALLOW_HEADERS", split=True, default=[]
+)
+
```
Comment (Coperh) [inline src/openklant/conf/base.py:505]: I notice that CORS_EXPOSE_HEADERS is in OpenZaak but not here nor in open-notificaties. Is that intentional?
```
CORS_EXPOSE_HEADERS = [
    "content-crs",
]
```

**Review by Coperh — APPROVED** [review]
Looks fine.

File: src/openklant/conf/base.py (hunk 1 above)
Comment (stevenbal) [reply src/openklant/conf/base.py:505]: This header is used by Open Zaak (and I think Objects) to determine the coordinate system in a geospatial context (https://docs.ogc.org/is/18-058/18-058.html#_coordinate_reference_system_information_independent_of_the_feature_encoding). Open Klant and Open Notificaties do not use this header, so the `CORS_EXPOSE_HEADERS` is not needed for that reason.

**Pull request merged**